File: java/trash/cipher3.java
public class LetterShiftExample {
    public static void main(String[] args) {
        String word = "Hello";
        int shift = 3; // Number of positions to shift each letter
        
        String shiftedWord = shiftLetters(word, shift);
        System.out.println("Shifted Word: " + shiftedWord);
    }
    
    public static String shiftLetters(String word, int shift) {
        StringBuilder shiftedWord = new StringBuilder();
        
        for (char ch : word.toCharArray()) {
            char shifted = (char) (ch + shift);
            shiftedWord.append(shifted);
        }
        
        return shiftedWord.toString();
    }
}

File: java/trash/cipher3.php
<html>
	<head>
		<title>Substitution Decryption</title>
	</head>
	<style>
	
	</style>
<body>
<h1>Decryption</h1>
<?php
function shiftLetters($word, $shift) 
{
    $shiftedWord = "";
    
    for ($i = 0; $i < strlen($word); $i++) 
	{
        $ch = $word[$i];
        $shifted = chr(ord($ch) + $shift);
        $shiftedWord .= $shifted;
    }
    
    return $shiftedWord;
}

$word = "PHYDQ#IHUQDQGHV";
$shift = -3;

$shiftedWord = shiftLetters($word, $shift);

echo "Cipher Text: " . $word;
echo "<br>";
echo("Plain Text: " . $shiftedWord);
echo "<br>";
echo "Key:3";
?>
</body>
</html>
